packages/browser/src/__tests__/cookieless.test.ts

```diff
@@ -93,6 +100,33 @@ describe('cookieless', () => {
             // should ignore cookie consent, and throw in test code due to logging an error
             expect(() => posthog.opt_in_capturing()).toThrow()
         })
+
+        it.each([[true], ['history_change']])(
+            'should send the initial pageview event when capture_pageview is %p',
```

Comment:
Hmm, somewhat misleading here because we're not varying cookieless_mode on this test, maybe improve the name?

```suggestion
            'should send the initial pageview event when cookieless_mode is "always" and capture_pageview is %p',
```